addons/core/functions/server/fnc_processRadiosServer.sqf

```diff
@@ -0,0 +1,57 @@
+/*
+    Name: TFAR_fnc_processRadiosServer
+
+    Author(s):
+        gandjustas
+
+    Description:
+        Handles (on server) getting switching radios, handles whether a manpack must be added to the player or not.
+
+    Parameters:
+        0: OBJECT - unit to replace radios
+
+    Returns:
+        Nothing
+
+    Example:
+       _unit call TFAR_fnc_processRadiosServer (with remoteExec from client)
+*/
+params ["_unit"];
+ if (alive _unit) then {
+    private _classes = _unit call TFAR_fnc_getDefaultRadioClasses;
+    private _lrRadio = _classes select 0;
+    private _personalRadio = _classes select 1;
+    private _riflemanRadio = _classes select 2;
+    private _defaultRadio = _riflemanRadio;
+    
+
+    //Handle backpack replacement for group leaders
+    if (leader _unit == _unit) then {
+        if (!TFAR_giveLongRangeRadioToGroupLeaders or {backpack _unit == "B_Parachute"} or {_unit call TFAR_fnc_isForcedCurator}) exitWith {};
+        if ([(backpack _unit), "tf_hasLRradio", 0] call TFAR_fnc_getVehicleConfigProperty == 1) exitWith {};
+
+        private _items = backpackItems _unit;
+        private _backPack = unitBackpack _unit;
+        _unit addBackpackGlobal _lrRadio;
```

Comment:
This won't work if the unit already has a backpack.

Reply:
It works even if unit already have a backpack. addBackpackGlobal  automaticaly puts exitsting backpack in front of soldier. Tested in game couple thothand times.

Reply:
>Adds a backpack for a unit. If a unit already has a backpack, the old backpack will be placed on the ground under the unit.
Why does TFAR have code then for manually putting the backpack on Ground -.-

Reply:
Just legacy code added long time ago and no one tried to improve.